lib/block-supports/align.php

```diff
@@ -17,10 +17,12 @@ function gutenberg_register_alignment_support( $block_type ) {
 			$block_type->attributes = array();
 		}
 
-		$block_type->attributes['align'] = array(
-			'type' => 'string',
-			'enum' => array( 'left', 'center', 'right', 'wide', 'full', '' ),
-		);
+		if ( ! array_key_exists( 'align', $block_type->attributes ) ) {
```

Comment:
Ah, yes!  Good addition we missed in the last one.  This makes sense to not define if it is already defined.